src/azul/indexer/__init__.py

```diff
@@ -50,25 +50,23 @@ class BundleFQID(SupportsLessThan):
     version: BundleVersion
 
 
-SOURCE_NAME = TypeVar('SOURCE_NAME', bound='SourceName')
+SOURCE_SPEC = TypeVar('SOURCE_SPEC', bound='SourceSpec')
 
 
-# FIXME: Rename to SourceSpec/SOURCE_SPEC, and all .name to .spec
-#        https://github.com/DataBiosphere/azul/issues/2843
 @attr.s(frozen=True, auto_attribs=True, kw_only=True)
-class SourceName(ABC, Generic[SOURCE_NAME]):
+class SourceSpec(ABC, Generic[SOURCE_SPEC]):
     """
     The name of a repository source containing bundles to index. A repository
     has at least one source. Repository plugins whose repository source names
     are structured might want to implement this abstract class. Plugins that
-    have simple unstructured names may want to use :class:`StringSourceName`.
+    have simple unstructured names may want to use :class:`StringSourceSpec`.
```

Comment:
```suggestion
    have simple unstructured names may want to use :class:`SimpleSourceSpec`.
```
There is no `StringSourceSpec`.